src/spinner/spinner.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { html, TemplateResult } from "lit";
```

Comment:
Suggestion: I think the eslint is set up in our project and you can format your codes in these changes.